package.json

```diff
@@ -230,34 +230,34 @@
   },
   "dependencies": {
     "args": "3.0.8",
-    "chalk": "2.3.2",
+    "chalk": "2.4.2",
     "color": "2.0.1",
     "columnify": "1.5.4",
-    "css-loader": "0.28.11",
+    "css-loader": "3.2.0",
     "got": "7.1.0",
     "json-loader": "0.5.7",
     "mousetrap": "chabou/mousetrap#useCapture",
-    "ms": "2.1.1",
+    "ms": "2.1.2",
     "opn": "5.3.0",
     "ora": "1.3.0",
-    "parse-url": "3.0.2",
+    "parse-url": "5.0.1",
     "php-escape-shell": "1.0.0",
-    "pify": "3.0.0",
-    "react": "16.2.0",
-    "react-deep-force-update": "2.1.1",
-    "react-dom": "16.2.1",
-    "react-redux": "5.0.7",
-    "recast": "0.13.0",
-    "redux": "3.7.2",
-    "redux-thunk": "2.2.0",
-    "reselect": "3.0.1",
-    "seamless-immutable": "7.1.3",
-    "semver": "5.5.0",
+    "pify": "4.0.1",
```

Comment:
this package `pify` should be moved to `app/package.json`

Reply:
@ivanwonder but lib/utils/paste.js uses it as well